tests/shape/rectangular_tag.lua

```diff
@@ -0,0 +1,13 @@
+local shape,cr,show = ... --DOC_HIDE
+ 
+shape.rectangular_tag(cr, 70, 70)
+show(cr) --DOC_HIDE
+ 
+-- shape.rectangular_tag(cr,20,70) --TODO broken
```

Comment:
I will add a DOC_HIDE for now, it has a minor visual issue, nothing terrible. I will fix it later when I improve the svg generator to show more than 3 images